src/VisualStudio/IntegrationTest/IntegrationTests/Workspace/WorkspacesNetCore2.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.Workspace
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class WorkspacesNetCore2 : AbstractEditorTest
+    {
+        protected override string LanguageName => LanguageNames.VisualBasic;
+
+        public WorkspacesNetCore2(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(WorkspacesNetCore2), WellKnownProjectTemplates.VisualBasicNetCoreClassLibrary)
+        {
+            EnableFullSolutionAnalysis();
+        }
+
+        [Fact(Skip = "VB Not Supported"), Trait(Traits.Feature, Traits.Features.Workspace)]
```

Comment:
So this has one test that's skipped...? :smile: Please revise to the tracking bug.